js/header.js

```diff
@@ -2,9 +2,29 @@
  * @file JS file for the header component.
  */
 
-(function headerScript(Drupal) {
+(function headerScript(Drupal, drupalSettings, once) {
   Drupal.behaviors.header = {
     attach(context) {
+      let mobileBreakpointJS = 768;
+      // If there is a drupalSettings variable for the breakpoint, use that.
+      if (
+        drupalSettings.localgov_base &&
+        drupalSettings.localgov_base.mobileBreakpointJS
+      ) {
+        let mobileBreakpoint = drupalSettings.localgov_base.mobileBreakpointJS;
+
+        // 1. If it has 'px' at the end, remove it.
+        mobileBreakpoint = mobileBreakpoint.replace('px', '');
+
+        // 2. Convert it to a number.
+        mobileBreakpoint = parseInt(mobileBreakpoint, 10);
+
+        // 3. Check that the value is greater than 0.
+        if (mobileBreakpoint > 0) {
+          mobileBreakpointJS = mobileBreakpoint;
+        }
+      }
+
```

Comment:
I think almost all of this could be handled in the existing `.theme` file (see below). By forcing the input to be numeric, _validating_ that it _is_ numeric, and adding a default value in the theme file, I think we could reduce this to e.g. `const mobileBreakpointJS = drupalSettings?.localgov_base?.mobileBreakpointJS ?? 768;`

(Even adding the 768 in js is probably overkill since it'd only be necessary when something fairly catastrophic had happened to `drupalSettings` or the `.theme` file).

Reply:
Done